reference/components/cf-monitord.markdown

```diff
@@ -13,12 +13,12 @@ available as [special variables][special variables] (e.g.
 [`$(mon.av_cpu)`][mon#mon.av_cpu]) to `cf-agent`, which may use them to inform
 policy decisions.
 
-`cf-monitord` keeps the promises made in `common`and `monitor` bundles, and is
-affected by  `common` and `monitor` control bodies.
+`cf-monitord` keeps the promises made in `common` and `monitor` bundles and is
+affected by `common` and `monitor` control bodies.
 
 **Notes:**
 
-* `cf-monitord` always considers the class ```monitor``` to be defined.
+* `cf-monitord` always considers the class `monitor` to be defined.
```

Comment:
Although it's not a big deal, this part seems to be intentional. The triple backticks tell the site generator to not make this a link automatically, which seems to make sense in this case.